### PR TITLE
Copied from trials

### PR DESCRIPTION
**Pull request by shahedakhatoon**
## Review from trials
Here I have done the following:
1. Copied the project from trials
2. Fixed linter errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    <title>Document</title>
+</head>
+<body>
+    |<header id="header">
+        <nav id="nav-bar">
+          <img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="logo" class="logo">
+          <ul>
+            <li class="nav-link" >Features</li>
+            <li class="nav-link" >How It Works</li>
+            <li class="nav-link" >Pricing</li>
```
Comment (shahedakhatoon) [inline index.html:17]: Please add a href link to all the items you have placed in the navbar. Adding links to the navbar items is essential for them to navigate onto the specific topics they refer to.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            <div>
+              <h2>Quality Assurance</h2>
+              <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the
+                pitch of your instrument.</p>
+            </div>
+          </div>
+        </section>
+        <section id="how-it-work">
+          <iframe id="video" src="https://www.youtube.com/embed/JEwOy5Fm1yw"></iframe>
+        </section>
```
Comment (shahedakhatoon) [inline index.html:59]: Please add a header(h2-h6) in this section. A section lacking a heading is not a good HTML coding practice.

**Review by shahedakhatoon — COMMENTED** [review]
# Changes required :recycle: 
1. Please check my inline comments.
2. Add a good description in your Readme.md file.

Submit for a re-review when you're done.

Thanks. :smile_cat:

**Review by keroloussamy — APPROVED** [review]
Hi @shahedakhatoon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- Include an explanation.✔️
- Comments are clear and easy to understand. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**